tox.ini

```diff
@@ -52,4 +54,6 @@ commands = /usr/bin/make -f Makefile.tox html
 [testenv:black]
 deps =
     black==20.8b1
+skip_install = true
+skipsdist = true
```

Comment:
@mbukatov could you provide some info about this change?

Reply:
As noted in the commit message, this prevents tox to run setuptools to create sdist tarball and to install ocs-ci. For source code checkers like black or flake8, full fledged installation with all dependencies is not necessary.

Moreover the implicit installation which happened behind the scenes during a tox run was not correct, as we mandate that ocs-ci should be installed via requirements.txt file, and that was not done there. For this PR, this caused these checks to fail, even though there were no issues. So instead of explicitly installing ocs-ci via requirements.txt file, I just removed this step altogether.